www/js/model.js

```diff
@@ -330,14 +345,19 @@ define([
 				// Summary: returns an array of all descriptions with
 				// name (label) and any associated node id (value).
 				// Note that the description may be empty.
-				// TO DO:  The list should be sorted.
+				// TODO:  The list should be sorted.
 				return array.map(this.getNodes(), function(node){
 					return {label: node.description, value: node.ID};
 				});
 			},
 			getDescription: function(/*string*/ id){
 				return this.getNode(id).description;
 			},
+			getAuthoredID: function(id){
+				// Summary: Return any matched given model id for student node.
+				var node = this.getNode(id);
+				return node && node.authoredID;
+			},
```

Comment:
Is this equivalent to descriptionID? if that is the case then authoredID won't exist in author model nodes. It would be part of student model nodes only. @riteshsamala is there something that I am missing here?